mvc/src/main/java/nextstep/mvc/DispatcherServlet.java

```diff
@@ -24,9 +20,11 @@
     private static final Logger log = LoggerFactory.getLogger(DispatcherServlet.class);
 
     private final List<HandlerMapping> handlerMappings;
+    private final List<HandlerAdapter> handlerAdapters;
```

Comment:
```suggestion
    private final HandlerMappings handlerMappings;
    private final HandlerAdapters handlerAdapters;
```
또는
```suggestion
    private final HandlerMappingRegistry handlerMappings;
    private final HandlerAdapterRegistry handlerAdapters;
```
와 같이 각각의 List를 관리하는 객체를 만들고 find 역할을 주는 것도 좋아보여요!

Reply:
일급 컬렉션! 크로플은 기본을 잊지 않고 계시군요👍
[반영](https://github.com/tributetothemoon/jwp-dashboard-mvc/commit/0e235789c3483522c7db4495d1d9488b5e118601)하였습니다. 좋은 피드백 감사합니다 :)